src/panels/config/script/ha-script-editor.ts

```diff
@@ -569,9 +569,13 @@ export class HaScriptEditor extends KeyboardShortcutMixin(LitElement) {
   }
 
   private _aliasChanged(alias: string) {
```

Comment:
Why would we calculate this on every key stroke and not when the user clicks save ?

Reply:
This was our current behavior. We can remove it but they would no longer see the entity ID be created while typing

Reply:
Oh, I didn't realize this was a thing. What weird that we show this to users 🤦 

For now this is ok as this fixes the bug then…